doc/man3/EVP_KEYEXCH_free.pod

```diff
@@ -58,6 +61,11 @@ all activated providers in the library context I<libctx>, and for each
 of the implementations, calls I<fn> with the implementation method and
 I<data> as arguments.
 
+EVP_KEYEXCH_gettable_ctx_params() and EVP_KEYEXCH_settable_ctx_params() return
+a constant B<OSSL_PARAM> array that describes the names and types of key
+parameters that can be retrieved or set by a key exchange algorithm using
+L<EVP_PKEY_CTX_get_params(3)> and L<EVP_PKEY_CTX_set_params(3)>.
```

Comment:
Furthermore, it's called by `EVP_PKEY_CTX_gettable_params()` (just like `EVP_KEYEXCH_settable_ctx_params()` is called by `EVP_PKEY_CTX_settable_params()`)...  and for most of the cases, the `EVP_PKEY_CTX` functions are more appropriate.  Makes me wonder if this doesn't over-do it.  ...  and yeah I get it, `openssl list` become quite complex without these.

(I'm not going to dig into this now...  maybe next week)